ceph.spec.in

```diff
@@ -30,15 +24,24 @@
 %bcond_with lowmem_builder
```

Comment:
I've just noticed the above line should probably be removed (given lowmem_builder is set one way or another in the blocks below)

Reply:
@tserong Good catch! Done.